ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -1433,7 +1438,8 @@ def neigh_lag_status_check(self):
         Ensure there are no interface flaps after warm-boot
         """
         for neigh in self.ssh_targets:
-            self.neigh_handle = Arista(neigh, None, self.test_params)
+            self.test_params['port_channel_intf_idx'] = [x['ptf_ports'][0] for x in self.vm_dut_map if x['mgmt_addr'] == neigh][0]
```

Comment:
`x['ptf_ports'][0]` will this always select first lag member, and not all?

Reply:
Fixed to take all lag members.